cmd/frontend/graphqlbackend/webhook_logs_test.go

```diff
@@ -279,8 +294,178 @@ func TestWebhookLogConnectionResolver_TotalCount(t *testing.T) {
 	})
 }
 
-func boolPtr(v bool) *bool           { return &v }
-func int32Ptr(v int32) *int32        { return &v }
-func int64Ptr(v int64) *int64        { return &v }
-func stringPtr(v string) *string     { return &v }
-func timePtr(v time.Time) *time.Time { return &v }
+func TestListWebhookLogs(t *testing.T) {
+	users := database.NewMockUserStore()
+	users.GetByCurrentAuthUserFunc.SetDefaultReturn(&types.User{SiteAdmin: true}, nil)
+
+	ctx := actor.WithActor(context.Background(), &actor.Actor{UID: 1})
+	webhookLogsStore := database.NewMockWebhookLogStore()
+	webhookLogs := []*types.WebhookLog{
+		{
+			ID:         1,
+			WebhookID:  int32Ptr(1),
+			StatusCode: 200,
+		},
+		{
+			ID:         2,
+			WebhookID:  int32Ptr(1),
+			StatusCode: 500,
+		},
+		{
+			ID:         3,
+			WebhookID:  int32Ptr(1),
+			StatusCode: 200,
+		},
+		{
+			ID:         4,
+			WebhookID:  int32Ptr(2),
+			StatusCode: 200,
+		},
+		{
+			ID:         5,
+			WebhookID:  int32Ptr(2),
+			StatusCode: 200,
+		},
+		{
+			ID:         6,
+			WebhookID:  int32Ptr(2),
+			StatusCode: 200,
+		},
+		{
+			ID:         7,
+			WebhookID:  int32Ptr(3),
+			StatusCode: 500,
+		},
+		{
+			ID:         8,
+			WebhookID:  int32Ptr(3),
+			StatusCode: 500,
+		},
```

Comment:
Nitpick, but: I'd save some precious bytes here 😛 
```suggestion
		{ID: 1, WebhookID: int32Ptr(1), StatusCode: 200},
		{ID: 2, WebhookID: int32Ptr(1), StatusCode: 500},
		{ID: 3, WebhookID: int32Ptr(1), StatusCode: 200},
		{ID: 4, WebhookID: int32Ptr(2), StatusCode: 200},
		{ID: 5, WebhookID: int32Ptr(2), StatusCode: 200},
		{ID: 6, WebhookID: int32Ptr(2), StatusCode: 200},
		{ID: 7, WebhookID: int32Ptr(3), StatusCode: 500},
		{ID: 8, WebhookID: int32Ptr(3), StatusCode: 500},
```

(Also makes it more readable, IMHO)